README.md

```diff
@@ -1314,6 +1314,7 @@ _See also [Ticketing](#ticketing), [Task management/To-do lists](#task-managemen
 - [Redmine](http://www.redmine.org/) - Redmine is a flexible project management web application. ([Demo](http://demo.redmine.org/), [Source Code](https://svn.redmine.org/redmine/)) `GPL-2.0` `Ruby`
 - [RhodeCode](https://rhodecode.com/) - RhodeCode is an open source platform for software development teams. It unifies and simplifies repository management for Git, Subversion, and Mercurial. ([Demo](https://try.rhodecode.com/), [Source Code](https://code.rhodecode.com/)) `AGPL-3.0` `Python`
 - [SCM Manager](https://www.scm-manager.org/) - The easiest way to share and manage your Git, Mercurial and Subversion repositories over http. ([Source Code](https://bitbucket.org/sdorra/scm-manager/src)) `BSD-3-Clause` `Java`
+- [sorcia](https://sorcia.org) - Sorcia is a minimalist web frontend for Git repositories with fine-grained user access controls, multiple themes to choose based on your taste and many more. ([Source Code](https://git.mysticmode.org/r/sorcia)) `AGPL-3.0` `Go`
```

Comment:
```suggestion
- [sorcia](https://sorcia.org) - Sorcia is a minimalist web frontend for Git repositories with fine-grained user access controls, multiple themes to choose based on your taste and much more. ([Source Code](https://git.mysticmode.org/r/sorcia)) `AGPL-3.0` `Go`
```

Or are there very many themes?

Reply:
No, there are only three themes at the moment. I should have worded it differently, when I say much more, it means more features that are there.

I'm in the process of implementing mailing lists into Sorcia and few others which I don't want to share it here at the moment. So, I though that "much more" would be appropriate.

Anyways, it doesn't matter much, I'll go with what you have suggested.

Reply:
Thing is - your approval process it too laggy! And I though I don't want to come back for changes.

Reply:
I just pointed out a grammatical mistake.

Reply:
Yes, I didn't take that offensive. I just think that the approval process is taking more time and I notified that.

As I said in the first message, I'm fine with your changes - we can go with that, if it is Okay with other members of this repository.